presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -134,6 +134,8 @@ public final class HiveSessionProperties
     public static final String SKIP_EMPTY_FILES = "skip_empty_files";
     public static final String LEGACY_TIMESTAMP_BUCKETING = "legacy_timestamp_bucketing";
 
+    public static final String STATS_BASED_FILTER_REORDER_DISABLED = "stats_based_filter_reorder_disabled";
```

Comment:
Do we want to add native prefix as the other native only session properties?

Reply:
Good point, but looks like the naming conversion only exists for system session property, connector specific ones we don't have.

Maybe we can add that in the follow up as long term solution